pkg/collector/corechecks/sbom/processor.go

```diff
@@ -3,18 +3,26 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2022-present Datadog, Inc.
 
+//go:build trivy
```

Comment:
Is it required? With the interfaces, I expect that `GetScanner` is just going to `panic` as it was never set.